dist/index.js

```diff
@@ -14,14 +14,21 @@ function _inherits(subClass, superClass) { if (typeof superClass !== "function"
 
 var React = require('react');
 
-var CaptchaScript = function CaptchaScript(cb) {
-  // Create script to initialize hCaptcha tool - hCaptcha
+var CaptchaScript = function CaptchaScript(cb, hl, endpoint) {
+  // Create script to init hCaptcha
   var script = document.createElement("script");
 
-  script.src = "https://hcaptcha.com/1/api.js?render=explicit";
+  window.hcaptchaOnLoad = cb;
+  script.src = "https://hcaptcha.com/1/api.js?render=explicit&onload=hcaptchaOnLoad";
   script.async = true;
 
-  script.addEventListener('load', cb, true);
+  if (typeof hl != "undefined" && hl != null) {
+    script.src += "&hl=" + hl;
+  }
+
+  if (typeof endpoint != "undefined" && endpoint != null) {
+    script.src += "&endpoint=" + endpoint;
```

Comment:
`endpoint` needs to be wrapped with `encodeURIComponent`, since its a query parameter.